docs/BreakingChanges/068 DbParameter.Precision and .Scale are now public vi.md

```diff
@@ -0,0 +1,35 @@
+## 68: DbParameter.Precision and .Scale are now public virt members instead of interface impls
```

Comment:
{
  "Id": "68",
  "Title": "DbParameter.Precision and .Scale are now public virt members instead of interface impls",
  "Details": "DbParameter.Precision and DbParameter.Scale are implemented as public virtual properties. They replace the corresponding explicit interface implementations, DbParameter.IDbDataParameter.Precision and DbParameter.IDbDataParameter.Scale.",
  "Suggestion": "When re-building an ADO.NET database provider, these differences will require the 'override' keyword to be applied to the Precision and Scale properties. This is only needed when re-building the components; exisiting binaries will continue to work.",
  "Link": "https://msdn.microsoft.com/en-us/library/dn458356%28v=vs.110%29.aspx#ADO",
  "Markdown": null,
  "ApplicableApis": [
    "P:System.Data.Common.DbParameter.Precision",
    "M:System.Data.Common.DbParameter.get_Precision",
    "M:System.Data.Common.DbParameter.set_Precision(System.Byte)",
    "P:System.Data.Common.DbParameter.Scale",
    "M:System.Data.Common.DbParameter.get_Scale",
    "M:System.Data.Common.DbParameter.set_Scale(System.Byte)"
  ],
  "Related": null,
  "VersionBroken": "4.5.1",
  "VersionFixed": null,
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "We can probably find this. But do we want to bother?\nSource analyzer status: Pri 3, Planned (code fix is possible)",
  "IsRetargeting": true,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] P:System.Data.Common.DbParameter.Precision
- [x] M:System.Data.Common.DbParameter.get_Precision
- [x] M:System.Data.Common.DbParameter.set_Precision(System.Byte)
- [x] P:System.Data.Common.DbParameter.Scale
- [x] M:System.Data.Common.DbParameter.get_Scale
- [x] M:System.Data.Common.DbParameter.set_Scale(System.Byte)